packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -135,6 +135,7 @@ abstract class SerializableFinder {
       case 'ByType': return ByType.deserialize(json);
       case 'ByValueKey': return ByValueKey.deserialize(json);
       case 'ByTooltipMessage': return ByTooltipMessage.deserialize(json);
+      case 'BySemanticLabel': return BySemanticLabel.deserialize(json);
```

Comment:
I believe the consensus is to use "semantics" as both noun and adjective throughout the code. There are only a couple of places where we use "semantic", which I believe @jonahwilliams is planning to clean up.

Reply:
Done